Project-Timer-frontend/src/app/login-page/login-page.component.html

```diff
@@ -0,0 +1,30 @@
+<form class="example-form" #loginForm="ngForm">
```

Comment:
staraj się stosować takie nazewnictwo alby dobrze opisywało daną zmienną lub funkcję, tyczy się to całego projektu, zmiennych, nazw funkcji, klas w html itd

Reply:
done